pkg/tracking/sentry.go

```diff
@@ -38,20 +38,22 @@ func (hook *Hook) Levels() []logrus.Level {
 // Fire uses the configured Sentry client to report the given Logrus Entry as
 // Sentry Event.
 func (hook *Hook) Fire(entry *logrus.Entry) error {
-	exceptions := []sentry.Exception{}
-
-	if err, ok := entry.Data[logrus.ErrorKey].(error); ok && err != nil {
-		stacktrace := sentry.ExtractStacktrace(err)
-		if stacktrace == nil {
-			stacktrace = sentry.NewStacktrace()
-		}
-		exceptions = append(exceptions, sentry.Exception{
-			Type:       entry.Message,
-			Value:      err.Error(),
-			Stacktrace: stacktrace,
-		})
+	entryError, ok := entry.Data[logrus.ErrorKey].(error)
+	if !ok || entryError == nil {
```

Comment:
[q]: since we are changing the semantics of the function, can we cover this in unit tests?